src/node.rs

```diff
@@ -44,34 +44,26 @@ pub struct PbftNode {
     /// Used for interactions with the validator
     pub service: Box<Service>,
 
-    /// Storage of state information
-    pub state: PbftState,
-
```

Comment:
Refactoring the codebase to pass state around instead of passing it as an instance variable isn't within the scope of the commit message and should be split into a separate commit. I am also not certain this is a good change, so it would be good to see some reasoning behind it.

Reply:
Moving `PbftState` into the `Storage` wrapper is pretty tied up with moving it into state managed by the engine. Would it make sense to just expand the commit description?